packages/adapter-neon/src/neon.ts

```diff
@@ -1,5 +1,5 @@
 /* eslint-disable @typescript-eslint/require-await */
-import type neon from '@neondatabase/serverless'
+import * as neon from '@neondatabase/serverless'
```

Comment:
The 1 line change?

Reply:
(Nope)

Reply:
Given we moved from `import * as pg from 'pg'` to `import pg from 'pg'`, shouldn't we do the same with `neon` as well?